src/main/resources/db/seed/R__02_location_docker_seed.sql

```diff
@@ -1,3 +1,5 @@
 call setup_prison_demo_locations('LEI', 'ITAG_USER');
 call setup_prison_demo_locations('BXI', 'ITAG_USER');
 call setup_prison_demo_locations('MDI', 'ITAG_USER');
+
+update prison_configuration set certification_approval_required = true;
```

Comment:
For docker env allow cert to be true on all

Reply:
Thanks for this, I've been manually setting this value to true in my frontend while building the new stuff